source/modules/task-list.js

```diff
@@ -60,51 +91,106 @@
     let editButton = document.createElement('button');
+    let doneButton = document.createElement('button');
 
-    // When user clicks on the task, it gets crossed off
-    taskButton.addEventListener('click', setFinished(taskContainer));
-    
     taskContainer.setAttribute('class', 'task');
     taskContainer.style.border = '3px solid black';
-    taskContainer.setAttribute('finished', 'false');
     
-    taskButton.setAttribute('type', 'button');
+    // When user clicks on the task, it gets crossed off
+    taskButton.setAttribute('type', 'radio');
+    taskButton.setAttribute('id', taskName);
+    taskButton.setAttribute('class', 'task-button');
+    taskButton.setAttribute('name', 'task-option');
+    taskButton.addEventListener('click', function(e) {
+        let currentTask = document.getElementById('current-task');
+            if (this.checked) {
+                currentTask.innerText = taskLabel.innerText;
+            }
+    });
 
-    taskButton.setAttribute('class', 'task-label');
-    taskButton.value = taskName;
-    taskLabel.contentEditable = false;
+    taskLabel.setAttribute('class', 'task-label');
+    taskLabel.setAttribute('for', taskName);
+    taskLabel.innerText = taskName;
 
     editButton.innerText = 'Edit';
+    doneButton.innerText = 'Done';
+    doneButton.style = "margin-left:50px";
 
-    // Let user edit the task's name when edit button is clicked
-    editButton.addEventListener('click', () => {
-        taskLabel.contentEditable = true;
-        taskLabel.focus();
-        taskLabel.addEventListener('keypress', (even) => { // turns off editability when hit enter key
-            if (even.key === 'Enter')
-                taskLabel.contentEditable = false;
-        });
+    // Check off task when complete
+    doneButton.addEventListener('click', () => {
+        if (taskButton.getAttribute('done') != 'true') {
+            taskButton.setAttribute('done', 'true');
+        } else {
+            taskButton.setAttribute('done', 'false');
+        }
+        
     });
 
-    taskButton.appendChild(taskLabel);
     taskContainer.appendChild(taskButton);
+    taskContainer.appendChild(taskLabel);
     taskContainer.appendChild(editButton);
+    taskContainer.appendChild(doneButton);
+    setEditTask(taskLabel, editButton);
     return taskContainer;
 }
 
 /**
- * @name clearTasksButton
+ * @name setEditTask
+ * @function
+ * @description Set up edit task
+ * @param taskLabel task name
+ * @param editButton edit button of the task
+ */
+function setEditTask(taskLabel, editButton) {
+    editButton.addEventListener('click', () => {
+        loadForm(taskLabel);
+    });
+}
+
+/**
+ * @name loadForm
+ * @function
+ * @description Load content form a task to task form
+ * @param content the content of the task
+ */
+function loadForm(content){
+    let taskName = document.getElementById("task-name");
+    document.getElementById("add-task-form").classList.remove("hidden");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9414373)
```suggestion
    document.getElementById('add-task-form').classList.remove("hidden");
```